docs/core/docker/docker-basics-dotnet-core.md

```diff
@@ -33,11 +33,11 @@ You can build both Windows and Linux containers with [multi-arch based tags](htt
 
 To complete this tutorial:
 
-#### .NET Core 2.0 SDK
+#### .NET Core 2.1 SDK
 
-* Install [.NET Core SDK 2.0](https://www.microsoft.com/net/core).
+* Install [.NET Core SDK 2.1](https://www.microsoft.com/net/core).
```

Comment:
wondering if sending them straight to downloads wouldn't be better:
https://www.microsoft.com/net/download